docs/setup-guide/scale-out-poc-setup.md

```diff
@@ -1,40 +1,27 @@
-### Setup proxy
+### Brief 
 
-Please follow [these steps](https://github.com/futurewei-cloud/arktos-perftest/wiki/How-to-setup-and-config-nginx-proxy-for-scale-out-design#setting-up-proxy). 
-
-The proxy can run on a machine of your choice. Make sure to update the ip addresses in the Nginx config for your partition servers.
-
-This is a one-time setup and pretty straightforward so it's not included in this PR as a script. 
-
-Will automate this in another PR.
-
-### Set proxy endpoint:
-
-This to to be done on ***both tenant and resource partitions***.
-
-Set the proxy url like
-
-```bash
-export SCALE_OUT_PROXY_ENDPOINT="http://192.168.0.120:8888"
-```
+This introduces the tool to set up the tenant-partition cluster and the resource-partition cluster.
 
 ### On the TENANT partition:
 
 Run 
 ```bash
-IS_RESOURCE_PARTITION=false arktos-up-scale-out-poc.sh
+IS_RESOURCE_PARTITION=false INSTALL_SCALE_OUT_PROXY=true SECOND_PARTITION_IP=[resource-partition-ip] hack/arktos-up-scale-out-poc.sh
```

Comment:
this gives the impression that the proxy can or should be installed on the tenant partition whereas it's not bound to any partition. I have it on my resource partition in local dev. 

I'd suggest keeping proxy setup doc in a separate section, and make it clear that if user wants to install it manually, they could still call the setup-proxy script without incurring the partition setup scripts.

Reply:
Per suggestion, I re-wrote the doc.